README.md

```diff
@@ -1,12 +1,12 @@
 <!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
-<a name="readme-top"></a>
+<a id="readme-top"></a>
```

Comment:
We may separately want to create a table of contents allowing you to jump *down* to sections (in addition to up).